paper.md

```diff
@@ -0,0 +1,77 @@
+We focused on making Turftopic first and foremost an easy-to-use library, that does not necessitate expert knowledge or excessive amounts of code to get started with, but gives great flexibility to power users.
+The library also includes three topic models, which to our knowledge only have implementations in Turftopic, these are: KeyNMF [@keynmf], S^3^ [@s3] and GMM.
+
+# Functionality
+
+Turftopic includes a wide array of contextual topic models from the literature, these include:
+FASTopic [@fastopic], Clustering models, such as BERTopic [@bertopic_paper] and Top2Vec [@top2vec], auto-encoding topic models, like CombinedTM [@ctm] and ZeroShotTM [@zeroshot_tm], KeyNMF [@keynmf], Semantic Signal Separation [@s3] and GMM.
+We believe these models to be representative of the state of the art in contextual topic modelling and intend to expand on them in the future.
+
+![Components of a Topic Modelling Pipeline in Turftopic](https://x-tabdeveloping.github.io/turftopic/images/topic_modeling_pipeline.png){width="800px"}
+
+Each model in Turftopic has an *encoder* component, which is used for producing continuous document-representations, and a *vectorizer* component, which extracts term counts in each documents, thereby dictating which terms will be considered in topics.
+The user has full control over what components should be used at different stages of the topic modelling process, thereby having fine-grained influence on the nature and quality of topics.
+
+The library comes loaded with a lot of utilities to help users interpret their results, including *pretty printing* utilities for exploring topics, *interactive visualizations* partially powered by the `topicwizard` [@topicwizard] Python package, and *automated topic naming* with LLMs.
+
+To accommodate a variety of use-cases, Turftopic can be used for dynamic topic modelling, where we expect topics to change over time, can be used for uncovering hierarchical structure in topics.
+Some models can also be fitted in an *online* fashion, where documents are accounted for as they come in by batches.
+Turftopic also includes *seeded* topic modelling, where a seed phrase can be used to retrieve topics relevant to the specific research question.
+
+# Target Audience
+
+Topic models can be utilized in a number of research settings, including exploratory data analysis, discourse analysis of many domains, such as newspapers, social media or policy documents.
+Turftopic has already been utilized by @keynmf for analyzing information dynamics in Chinese Diaspora Media, and is currently being used in multiple ongoing research projects, including one analyzing discourse on the HPV vaccine in Denmark.
+In addition, the library has already seen extensive use in student exam papers at Aarhus University.
+We expect that computational researchers, especially in digital humanities and in social sciences will benefit from an easy-to-use, unified contextual topic modelling package.
```

Comment:
I would move this to the beginning (target audience -> more specific use setting -> exact usage with citations). 

I would also make sure to include industry here

(this section is as much as who we want to target not who are currently using it)